static/components/PoiCard.js

```diff
@@ -9,8 +9,10 @@ class PoiCard extends React.Component {
     super(props)
     this.state = {
       from: {},
-      to: {}
+      to: {},
     }
+    this.hoverCard = this.hoverCard.bind(this)
```

Comment:
I would put that above line 24 to remind what that relates to.
The idea is that these line of code don't talk by themselves. I don't feel like someone coming on there would know what that's used for. Or even ourselves in a month from now.
How do you feel about it?